src/vertex.ml

```diff
@@ -18,6 +18,10 @@ functor
     module Page = Store.Page
     module Header = Vertex_header.Make (Params) (Store) (Common)
 
+    module Phantom = Field.MakeBool (struct
+      let size = 1
+    end)
+
```

Comment:
In `Vertex_header` we have `Ndeadentry`. I think `Phantom` should be called something like `IsDead` or `Isdead` for consistency.